client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -80,6 +88,7 @@
     private         ObjectMapper                objectMapper;
     private         int                         timeout = 10;
     private         TimeUnit                    timeoutUnit = TimeUnit.SECONDS;
+    private final Map<Class<?>, List<Tuple<Function<Object, Object>, Annotation[]>>> beanParamCache = new HashMap<>();
```

Comment:
What is the reason for adding a Tuple instead of having a proper class with names that explain what the values represent?

Reply:
Because adding a proper class is painful? I guess you're not a fan of the Tuple concept...

Reply:
Aha...